component/xrds.libsonnet

```diff
@@ -57,6 +57,7 @@ local loadCRD(crd, tag) = std.parseJson(kap.yaml_load(inv.parameters._base_direc
 local xrdFromCRD(name, crd, defaultComposition='', connectionSecretKeys=[]) =
   kube._Object('apiextensions.crossplane.io/v1', 'CompositeResourceDefinition', name) + common.SyncOptions + {
     spec: {
+      defaultCompositionUpdatePolicy: if name == 'xobjectbuckets.appcat.vshn.io' then 'Automatic' else if params.deploymentManagementSystem.enabled then 'Manual' else 'Automatic',
```

Comment:
It might make sense to write a short comment what and why this nested if clause exists otherwise it might be hard to understand this in a couple of months